llvm/test/CodeGen/RISCV/GlobalISel/float-zfa.ll

```diff
@@ -0,0 +1,161 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+
+; RUN: llc -mtriple=riscv32 -mattr=+zfa -global-isel < %s \
+; RUN: | FileCheck %s --check-prefixes=CHECK-ZFA-f32
+; RUN: llc -mtriple=riscv64 -mattr=+zfa -global-isel < %s \
+; RUN: | FileCheck %s --check-prefixes=CHECK-ZFA-f32
+; RUN: llc -mtriple=riscv32 -mattr=+f -global-isel < %s \
```

Comment:
We should already have coverage without Zfa in float-intrinsics.ll

Reply:
Removed the intrinsic check here